scRNA-seq-advanced/02-celltype_assignment.Rmd

```diff
@@ -44,149 +49,233 @@ set.seed(12345)
 
-CD3 is a marker of T-cells 
+## Investigating cell types
 
-```{r}
-scater::plotUMAP(sce, 
-                 colour_by = "CD3")
+### Using cell surface marker data
+
+The first way we will work on to identify the cell types of the individual cells in our sample is to work with the ADT markers that are part of this data set. 
+These antibody markers were (hopefully) chosen for their relevance to the cell population we are working with.
+
+The first marker we will look at is `CD3`, which is a protein complex that is found on the surface of T cells.
+We can plot where this is found using the same `plotUMAP()` function as before. 
+
+Note that the function is able to look for plotting data in fields found in the `colData` object (as we used it above), in the gene expression matrix (as we used it in the previous notebook), *AND* in `altExp` matrices!
+So all we need to do to color by the ADT counts (as normalized into the `logcounts` matrix) is give the name of the tag that we want to plot in the `colour_by` argument.
+
+```{r plot CD3, live = TRUE}
+# plot CD3 expression
+scater::plotUMAP(sce, colour_by = "CD3")
 ```
 
-CD4+ T-cells
+It appears that we have a number of potential T cells down in the lower left!
 
-```{r}
+Let's look at a couple of other markers to try to break those up more specifically.
+
+Two other markers of relevance to the T cells are `CD4` and `CD8`.
+The `CD4` complex is present in helper T cells (hence their other common name, CD4+ T cells).
+By contrast, the `CD8` complex is found on killer T cells (CD8+ cells).
+
+Let's plot the ADT results for those two markers as well below:
+
+```{r plot CD4, live=TRUE}
+# plot CD4 marker
 scater::plotUMAP(sce, 
                  colour_by = "CD4")
 ```
-CD8+ T-cells
 
-```{r}
+```{r plot CD8, live=TRUE}
+# plot CD8 marker
 scater::plotUMAP(sce, 
                  colour_by = "CD8")
 ```
 
 
-So one thing we could do is construct a set of rules based on ADT tags, similar to how you might set gating rules for cell sorting.
+### Rule-based classification
 
+When we have markers like these, with clear and established cell types, we might want to start our classification by working with these markers alone.
+Such classification might be considered as analogous to a cell-sorter assay, where we would set up some rules to look at a few markers for each cell and use those to assign a cell type classification.
 
-What thresholds should we use?
-If this is a useful marker, we should see a bimodal plot of the marker density.
+To do this, we will need to make some decisions, like what thresholds should we use to determine whether a cell is or is not expressing a particular marker. 
+In general, if a marker is going to be useful for this type of assignment, we would like to see a bimodal plot for the marker in question that can be used to separate the population into two groups of cells based on the level of that marker.
```

Comment:
```suggestion
In general, markers that are useful for this cell-typing approach would have a bimodal distribution of expression levels which can be used to separate the population into two groups of cells.
```
Add a second sentence here about what the bimodal means here (foreground/background).

Reply:
I would echo the explanation here about first defining what a "rule" is to assign a cell type and then introducing the idea that although a gene could be a marker for a set of cells it's not going to be binary across cells, there is going to be some level of background and how do you define what that that background is.